macros/ano_macros.cfg

```diff
@@ -28,12 +28,7 @@
     [/filter]
 #enddef
 
-#define SCROLL_TO X Y
-    [scroll_to]
-        x={X}
-        y={Y}
-    [/scroll_to]
-#enddef
+# (SCROLL_TO is a mainline macro; no need to define it ourselves)
```

Comment:
Just delete, no need for comment.

Reply:
ok, removed in [a445cf8](https://github.com/nemaara/A_New_Order/pull/163/commits/a445cf8f1e72dd6801aa9756cd932b647ecb4170).